server/channels/db/migrations/postgres/000138_add_remoteid_channelid_to_post_acknowledgements.up.sql

```diff
@@ -0,0 +1,12 @@
+ALTER TABLE postacknowledgements ADD COLUMN remoteid VARCHAR(26) DEFAULT '';
+ALTER TABLE postacknowledgements ADD COLUMN channelid VARCHAR(26) NOT NULL DEFAULT '';
+
+UPDATE postacknowledgements pa
+SET channelid = p.channelid
+FROM posts p
+WHERE pa.postid = p.id
+AND pa.channelid = '';
+
+ALTER TABLE postacknowledgements ALTER COLUMN channelid DROP DEFAULT;
+
+CREATE INDEX idx_post_acknowledgements_postid_remoteid ON postacknowledgements(postid, remoteid);
```

Comment:
`CREATE INDEX` always needs to run concurrently. See `postgres/000118_create_index_poststats.up.sql` as an example.

Reply:
removed the INDEX